docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -11,24 +11,24 @@ Purpose
 
 **HTTP flow**
 
-* SDL sends the PTS snapshot as binary data via an `OnSystemRequest` mobile request from the system to the backend. Both the "url" that the PTS will be forwarded to and the "timeout" must be taken from the Local Policy Table.
-    * If no "url" is provided in the Local Policy Table, it is supposed that the mobile application will send the Policy Table Update data back to SDL.
+* SDL sends the Policy Table Snapshot (PTS) as binary data via an `OnSystemRequest` mobile request from the system to the back-end. Both the `url` that the PTS will be forwarded to and the `timeout` must be taken from the Local Policy Table.
+    * If no `url` is provided in the Local Policy Table, it is supposed that the mobile application will send the Policy Table Update data back to SDL.
 
-`RequestType` defines the type of the requested data from a mobile device or the cloud. The HMI may request this data, but it's SDL's responsibility to block a `SystemRequest` in case the request intends to transfer a data type not allowed by the policy table.  
+`RequestType` defines the type of the requested data from a mobile device or the cloud. The HMI may request this data, but it is SDL's responsibility to block a `SystemRequest` in case the request intends to transfer a data type not allowed by the Policy Table.  
 `requestSubType` is filled for supporting OEM proprietary data exchanges.
 
-The HMI is informed about `requestTypes`, `requestSubType` that are allowed by policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
+The HMI is informed about `requestTypes`, `requestSubType` that are allowed by Policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
 
 !!! NOTE
 
-If the HMI sends `OnSystemRequest` with a request type disallowed by the policy table, SDL will ignore it.  
-In case PT contains some value for `requestSubType` param and the HMI sends `OnSystemRequest` with `requestSubType=<not_in_PT>`, SDL does not forward this notification to mobile app. 
+If the HMI sends `OnSystemRequest` with a request type disallowed by the Policy Table, SDL will ignore it.  
+In case Policy Table contains some value for `requestSubType` param and the HMI sends `OnSystemRequest` with `requestSubType=<not_in_PT>`, SDL does not forward this notification to mobile app. 
```

Comment:
```suggestion
In case the Policy Table contains some value for `requestSubType` param and the HMI sends `OnSystemRequest` with `requestSubType=<not_in_PT>`, SDL does not forward this notification to mobile app. 
```

Reply:
fixed in 4418b048476f572006f760686c7f9a2a0db85da6